content/doc/administration/requirements/upgrade-java-guidelines.adoc

```diff
@@ -29,15 +29,11 @@ If you need to upgrade Jenkins as well as the JVM, we recommend that you:
   ** Use a package manager to install the new JVM.
   ** Make sure the default JVM is the newly installed version. If it is not, run `systemctl edit jenkins` and set either the `JAVA_HOME` environment variable or the `JENKINS_JAVA_CMD` environment variable.
 
-If you use Docker containers to run Jenkins, the default Docker containers use Java 11 beginning with link:/doc/upgrade-guide/2.303/#upgrading-to-jenkins-lts-2-303-1[Jenkins 2.303.1].
-If you need to remain with Java 8 in a Docker container, append `-jdk8` to the Docker image tag.
-
-Starting with the 2.357 weekly release, Java 11 is required for weekly releases.
-These instructions will still apply to the Long Term Support release line until September 2022, when the LTS release line will mirror the weekly release line Java requirements.
+Starting with the 2.357 weekly release and Long Term Support release 2.361.1, Java 11 is required.
```

Comment:
For consistency with your changes to line 16 of `java.adoc`:

```suggestion
Starting with the Jenkins 2.357 weekly release and the Long Term Support release 2.361.1, Java 11 or Java 17 is required.
```

Reply:
Definitely agree here, and I'll make that update